$NetBSD$

AFAIKT, fileno() should be in the std namespace.  It is
on QNX, global everywhere else?

--- configure.orig	2005-09-03 10:28:36.000000000 -0400
+++ configure
@@ -6377,6 +6377,9 @@ cat >>conftest.$ac_ext <<_ACEOF
 
 #include <stdio.h>
 extern "C" { void fileno(int); }
+#ifdef __QNXNTO__
+extern "C" { namespace std { void fileno(int); } }
+#endif
 
 
 int
